addon/services/infinity.js

```diff
@@ -296,6 +276,24 @@ export default Service.extend({
     scheduleOnce('afterRender', this, 'infinityModelLoaded', { totalPages: totalPages });
   },
 
+  /**
+    hook to modify results from response
+
+    @private
+    @method _afterInfinityModel
+   */
+  _afterInfinityModel(newObjects, infinityModel) {
+    if (!this.afterInfinityModel || typeof this.afterInfinityModel !== 'function') {
+      return newObjects;
+    }
+
+    let result = this.afterInfinityModel(newObjects, infinityModel);
+    if (result) {
+      return result;
+    }
+    return newObjects;
+  },
```

Comment:
We are moving towards a service based approach; however, this is to fix backwards compat support.